projects/canopy/src/lib/card/card.notes.ts

```diff
@@ -112,6 +175,9 @@ Generate the markup as show in the example below, no current modifiers.
     <div class="lg-card-content">
         Your content
     </div>
+    <div class="lg-card-footer">
+        Your content
```

Comment:
```suggestion
        Your footer
```
Minor, but I guess the original 'Your content' was written to match the fact that it was the content component.

Reply:
Done.